syntax = "proto2";
import "response.proto";

message Response_ForgotPasswordRequest {
    extend Response {
        optional Response_ForgotPasswordRequest ext = 1016;
    }
    optional bool challenge_email = 1;
}
